rules/windows/registry/registry_set/registry_set_persistence_globalflags.yml

```
@@ -8,9 +8,9 @@ description: Detects registry persistence technique using the GlobalFlags and Si
 references:
     - https://oddvar.moe/2018/04/10/persistence-using-globalflags-in-image-file-execution-options-hidden-from-autoruns-exe/
     - https://www.deepinstinct.com/2021/02/16/lsass-memory-dumps-are-stealthier-than-ever-before-part-2/
-author: Karneades, Jonhnathan Ribeiro, Florian Roth
+author: Karneades, Jonhnathan Ribeiro, Florian Roth, Swachchhanda Shrawan Poudel
```

You don't add yourself as the author just for fixing a typo :) that's not how it works
```suggestion
author: Karneades, Jonhnathan Ribeiro, Florian Roth
```